panther_analysis_tool/main.py

```diff
@@ -316,6 +191,47 @@ def _convert_keys_to_lowercase(mapping: Dict[str, Any]) -> Dict[str, Any]:
     return {k.lower(): v for k, v in mapping.items()}
 
 
+def zip_analysis_chunks(args: argparse.Namespace) -> List[str]:
+    logging.info("Zipping analysis items in %s to %s", args.path, args.out)
+
+    current_time = datetime.now().isoformat(timespec="seconds").replace(":", "-")
+    zip_chunks = [
+        # note: all the files we care about have an AnalysisType field in their yml
+        # so we can ignore file patterns and leave them empty
+        ZipChunk(patterns=[], types=(DATAMODEL, RULE, POLICY, PACK, GLOBAL)),  # type: ignore
+        ZipChunk(patterns=[], types=(QUERY, SCHEDULED_RULE)),  # type: ignore
+        ZipChunk(patterns=[], types=LOOKUP_TABLE),  # type: ignore
```

Comment:
Do we still need this param?

Reply:
yeah we're still using it when no chunking is set, we just do a * pattern on a single chunk